module TestFailFunPhantomReturn {
   err X;

   risky <.X> fun TestFunApply[phantom A]():(a:A) {
      throw .X;
   };
}